### PR TITLE
An update to episode 'collaborating'

### PR DESCRIPTION
**Pull request by anenadic**
Includes:

- a paragraph on meeting notes as communication means
- a note on accessibility issues when using colours
- a small typo fix

### CODE REVIEW

File: episodes/24-collaborating.md [hunk 1]
```diff
@@ -82,15 +82,15 @@ GitHub provides template boards that by default contain the three ‘basic’ co
...
-- **Labels**: if you convert each card into an issue, then you can label them with their priority - remember GitHub lets you create custom labels and set their colours. Labels can provide a very visually clear indication of issue priority but require more administrative work on the project, as each card has to be an issue to be assigned a label.
+- **Labels**: if you convert each card into an issue, then you can label them with their priority - remember GitHub lets you create custom labels and set their colours. Label colours can provide a very visually clear indication of issue priority but require more administrative work on the project, as each card has to be an issue to be assigned a label. You also need to be aware of accessibility issues for colour-blind people if choosing this route for issue prioritisation.
```
Comment (anenadic) [inline episodes/24-collaborating.md:93]:
```suggestion
- **Labels**: if you convert each card into an issue, then you can label them with their priority - remember GitHub lets you create custom labels and set their colours. Label colours can provide a very visually clear indication of issue priority but require more administrative work on the project, as each card has to be an issue to be assigned a label. If you choose this route for issue prioritisation - be aware of accessibility issues for colour-blind people when picking colours. 
```

File: episodes/24-collaborating.md [hunk 2]
```diff
@@ -68,7 +68,7 @@ As we have seen in the previous episode, GitHub's notifications framework **Ment
...
 
-TODO: meeting minutes to provide records of meetings and communicate decisions. Do we want to cover this? 
+Meeting minutes are a useful way of providing a permanent record of the purpose of a meeting and what was talked about, including any decisions made or actions taken, that can be referred back to for any follow-ups. Recording meeting minutes does not have to be hard - remember that you are part of a team and there are many platforms that allow collaborative note taking. The Carpentries provide [CodiMD](https://codimd.carpentries.org/) and [Etherpad](https://pad.carpentries.org/) instances as two options for taking shared notes, widely used at Carpentries workshops for sharing pieces of code, references and notes between insturcors and learners. For some advice on holding effective meetings, see the [“Meetings, meetings, meetings”](http://teachtogether.tech/en/index.html#s:meetings) section of [Teaching Tech Together](http://teachtogether.tech/en/index.html) by Greg Wilson.
```
Comment (tobyhodges) [inline episodes/24-collaborating.md:71]:
```suggestion
Meeting minutes are a useful way of providing a permanent record of the purpose of a meeting and what was talked about, including any decisions made or actions taken, that can be referred back to for any follow-ups. Recording meeting minutes does not have to be hard - remember that you are part of a team and there are many platforms that allow collaborative note taking. The Carpentries provides [CodiMD](https://codimd.carpentries.org/) and [Etherpad](https://pad.carpentries.org/) instances as two options for taking shared notes, widely used at workshops for sharing pieces of code, references and notes between instructors and learners. For some advice on holding effective meetings, see [the _Meetings, meetings, meetings_ section of Teaching Tech Together](http://teachtogether.tech/en/index.html#s:meetings) by Greg Wilson.
```

File: episodes/24-collaborating.md [hunk 3]
```diff
@@ -100,7 +100,7 @@ The moment your collaborator group exceeds 3 or 4 members (and fewer if the coll
 
 Here are some aspects of governing a project that you should consider. These are borrowed from [the _Working in teams_ chapter](https://merely-useful.tech/py-rse/teams.html) of [_Research Software Engineering with Python_](https://merely-useful.tech/py-rse/index.html), a book on how to work productively in small teams where everyone is welcome:
 
-- [establish a Code of Conduct](https://merely-useful.tech/py-rse/teams.html#teams-coc)
+- [establishing a Code of Conduct](https://merely-useful.tech/py-rse/teams.html#teams-coc)
```
Comment (tobyhodges) [inline episodes/24-collaborating.md:103]:
```suggestion
- [Codes of Conduct](https://merely-useful.tech/py-rse/teams.html#teams-coc)
```

I think this is more consistent with the phrasing of the other points in the list.

**Review by tobyhodges — CHANGES_REQUESTED** [review]

**Review by tobyhodges — APPROVED** [review]

**Comment by anenadic** [comment]
Looks like some tests are failing atm @tobyhodges - not sure if this can be safely ignored or not, so postponing the merge until we are sure.

**Comment by tobyhodges** [comment]
Tests are failing on all PRs right now, and the failures seem to be unrelated to the changes in the PR. Local builds also work fine. So I think the failures are spurious. 

I am happy to merge this now and take responsibility for whatever terrible consequences that brings!

**Comment by anenadic** [comment]
> Tests are failing on all PRs right now, and the failures seem to be unrelated to the changes in the PR. Local builds also work fine. So I think the failures are spurious.
> 
> I am happy to merge this now and take responsibility for whatever terrible consequences that brings!

Sounds good.

**Pull request merged**